docs/src/app/og/route.tsx

```diff
@@ -29,104 +29,14 @@ export async function GET(req: NextRequest) {
           display: 'flex',
           height: '100%',
           width: '100%',
+          backgroundImage:
+            'url(https://thinkmill-labs.keystatic.net/keystatic-site/images/km9uvzfb8fqo/og-image-bg)',
```

Comment:
This background image doesn't seem to be used? so the background is just the colour? I did some investigation and it seems like it's because of https://github.com/vercel/satori/issues/273 and Keystatic Cloud serving webp by default, you should be able to fix that by using `https://thinkmill-labs.keystatic.net/keystatic-site/images/km9uvzfb8fqo/og-image-bg?format=png` (from testing that seemed to solve the error from webp but the image still wasn't actually being displayed)

Reply:
Hey Emma, thanks for looking into this. I had a bit of a look and test a few ideas from what you've come back with, with no success. I'm not sure why it's not loading up the background image, as it seemed to work fine from the implementation before the brand update work!?

The previous code for the background image in the original version was this:
`backgroundImage: 'url(https://thinkmill-labs.keystatic.net/keystatic-site/images/nt24l5xes6jj/open-graph-bg)',`

Reply:
From checking out the commit before the brand updates (183e1293df2871f19e1239965a19f29e2b700846) and trying it now, it seems as though it broke. I'm guessing that would have broke when we added image transforms to `*.keystatic.net` 😬  (though adding `?format=png` to the url in 183e1293df2871f19e1239965a19f29e2b700846 does seem to work)

Reply:
Just pushed a commit that seems to fix it, it seems like having an image bigger than the image that's being created made something decide to not render it so resizing the image to be the same size as the output worked